doc/admin-guide/plugins/ts_lua.en.rst

```diff
@@ -178,17 +178,18 @@ Here is an example:
 
 ts.debug
 --------
-**syntax:** *ts.debug(MESSAGE)*
+**syntax:** *ts.debug(TAG, MESSAGE)*
```

Comment:
you are making this backward compatible in the code. Perhaps we can state that in the doc.